superset/databases/decorators.py

```diff
@@ -55,7 +55,7 @@ def wraps(
             database, Table(table_name_parsed, schema_name_parsed)
         ):
             stats_logger_manager.instance.incr(
-                f"permisssion_denied_{self.__class__.__name__}.select_star"
+                f"permission_denied_{self.__class__.__name__}.select_star"
```

Comment:
changing this value can affect people's metrics if they actively watching this value. That said, we don't have strict semver guidelines around logging and breaking changes, and I don't think that this value is used much. I'm probably ok with letting this slip through, but curious if anyone else has concerns. We can also wait until 3.0, but you'll likely have merge conflicts with these files if you wait that long to merge. Or you can just remove this change.